pkg/actuators/machine/actuator.go

```diff
@@ -408,34 +411,34 @@ func (a *Actuator) Update(context context.Context, cluster *machinev1.Cluster, m
 // Exists determines if the given machine currently exists. For AWS we query for instances in
 // running state, with a matching name tag, to determine a match.
 func (a *Actuator) Exists(context context.Context, cluster *machinev1.Cluster, machine *machinev1.Machine) (bool, error) {
-	glog.Info("checking if machine exists")
+	glog.Info("Checking if machine exists")
 
 	instances, err := a.getMachineInstances(cluster, machine)
 	if err != nil {
-		glog.Errorf("error getting running instances: %v", err)
+		glog.Errorf("Error getting running instances: %v", err)
 		return false, err
 	}
 	if len(instances) == 0 {
-		glog.Info("instance does not exist")
+		glog.Info("Instance does not exist")
 		return false, nil
 	}
 
 	// If more than one result was returned, it will be handled in Update.
-	glog.Infof("instance exists as %q", *instances[0].InstanceId)
+	glog.Infof("Instance exists as %q", *instances[0].InstanceId)
 	return true, nil
 }
 
 // Describe provides information about machine's instance(s)
 func (a *Actuator) Describe(cluster *machinev1.Cluster, machine *machinev1.Machine) (*ec2.Instance, error) {
-	glog.Infof("checking if machine exists")
+	glog.Infof("Checking if machine exists")
```

Comment:
some of these could go into different level https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md